### PR TITLE
trigger warning when browser dependent CDN was detected

### PR DESCRIPTION
**Pull request by kota-yata**
Display warning if the pasted domain is matched with the constant list (```URL_LIST```) in ```checkBrowserDepedency```, which currently contains only fonts.googleapis.com.
Screen Shot 2021-12-03 at 0 20 37

### CODE REVIEW

**Comment by kota-yata** [comment]
Related to #514

**Comment by kota-yata** [comment]
Preview : https://kota-yata.github.io/srihash.org/

File: app.js [hunk 1]
```diff
@@ -73,6 +73,19 @@ async function integrityMetadata(text, algorithm) {
...
 
+function checkBrowserDependency(url) {
+  const URL_LIST = [
+    'fonts.googleapis.com'
+  ];
+  const [, domain] = url.match(/^https?:\/\/([^/?#]+)(?:[/?#]|$)/i);
+
+  if (URL_LIST.find((e) => e === domain)) {
+    return [true, domain];
+  }
```
Comment (mozfreddyb) [inline app.js]: Please use the bulit-in `URL()`  parser API. E.g.,
```js
let hostname = new URL(url).hostname;
if URL_LIST.includes(hostname) { 
// ...
```

File: style.css [hunk 2]
```diff
@@ -192,6 +192,14 @@ footer {
   background-color: #ffb6c1;
 }
 
+#warning {
+  background: #d5e47f;
```
Comment (mozfreddyb) [inline style.css]: Not a huge fan of the color, but I'm also not known for my good taste. Maybe something more like a warning-orange? :) It's OK to leave this if you feel strong.y

**Review by mozfreddyb — COMMENTED** [review]
Thanks for the quick pull request!  Please switch to using the URL parsing API and we're good to go.

File: style.css (hunk 2 above)
Comment (kota-yata) [reply style.css]: I made it little more vivid in https://github.com/mozilla/srihash.org/pull/518/commits/8698f0dc093d7b0d24042ac421444a6820d87170 . Yellow looks better to me as a warning color than orange. Bright yellow is also better for accessibility.

File: app.js (hunk 1 above)
Comment (kota-yata) [reply app.js]: fixed

**Review by mozfreddyb — CHANGES_REQUESTED** [review]
Thanks! I found another bug, though. :)
We'll get more warning boxes, if you keep hitting the "hash" button.
You need to either remove the box unconditionally (before the `if (isBrowserDependent)...`) or better, modify in-place if it already exists.

P.S: Thank you for working on the color.

**Comment by kota-yata** [comment]
Probably fixed in https://github.com/mozilla/srihash.org/pull/518/commits/3c8c796523e43cae7d7ffff83475ca6ba3fc5c06

**Review by mozfreddyb — APPROVED** [review]

**Comment by mozfreddyb** [comment]
thank you!

**Pull request merged**